_rules/text-contrast-afw4f7.md

```diff
@@ -139,12 +139,10 @@ This 14pt bold black text has a contrast ratio of 3.6:1 on the gray background.
 
 #### Passed Example 7
 
-This text does not convey anything in human language.
+Despite falling below the minimum contrast ratio, the content of the button ("X") does not convey anything in human language; as a result, it complies with this rule. However, [Success Criterion 1.4.11 Non-Text Content][sc1411] should be examined for this non-text element.
```

Comment:
**Suggestion (non-blocking):** I'd rather swap the sentence around to keep front-loading the target (and reason).
```suggestion
The content of this button ("X") does not convey anything in human language. Due to its low contrast, it is not satisfying [Success Criterion 1.4.11 Non-Text Content][sc1411], but this rule doesn't consider it.
```

(also for the other rule)

Reply:
It's currently passing 1.4.11 since the contrast ratio is above 3:1. My idea was to remember the author to verify if 1.4.11 is passing or not, without clearly saying if it's passing or not (since it is not a matter of this specific rule).

What about: 

The content of this button ("X") does not convey anything in human language; therefore, it passes the rule regardless of its contrast ratio. However, [Success Criterion 1.4.11 Non-Text Content][sc1411] should be examined for this non-text element.

?

Reply:
Perfect.